api/services/ocean/aws/paths/cluster-roll.yaml

```diff
@@ -18,7 +18,7 @@ put:
       example: o-1abcd124
       description: >
         The ID of the Ocean
-    - in: path
+    - in: query
```

Comment:
wrong change.
the URL is https://api.spotinst.io/ocean/aws/k8s/cluster/{OCEAN_ID}/roll/{**ROLL_ID**}